clients/cobol-lsp-vscode-extension/src/services/Settings.ts

```diff
@@ -119,7 +113,7 @@ export class SettingsService {
     private static evaluateVariable(dataList: string[], variable: string, value: string): string[] {
         const result: string[] = [];
         if (dataList) {
-            dataList.forEach(d => result.push(d.replace("$" + variable, value)))
+            dataList.forEach(d => result.push(d.replace(`$\{${variable}}`, value)))
```

Comment:
Seems that you need to escape the second brace like this: `$\{${variable}\}`.
But it works as it is...

Reply:
Fixed, anyway